### DISCUSSION TITLE
Add comparison color name

### DISCUSSION THREAD

**Discussion by Oubaluna** [saadeghi/daisyui#2810] [Ideas / feature requests]
Hi there,

I'm loving using daiyui and I just encounter something that would be nice to implement : two color class name for comparison stuff like a comparison-1 and a comparison-2 color for each theme that would make a nice colorful comparison between stuff:)

don't hesitate to tell me if it's a dumb idea and if you would know a work around that would work for most theme

**Comment by saadeghi** [comment]
Can you explain more about the use case please? I'm not sure if I goy what you mean.
